unitTests/CommunicationPattern.C

```diff
@@ -24,75 +24,264 @@ protected:
 };
 
 
-TEST_F(CommunicationPatternFixture, can_get_comm_size)
+TEST_F(CommunicationPatternFixture, compute_owner_rank_single_owner)
 {
+    // Arrange
     auto comm = exec->get_gko_mpi_host_comm();
-    EXPECT_EQ(comm->size(), 4);
```

Comment:
we still need a test if comm->size() works and returns a result that is > 1

Reply:
Okay, but it is better to put this test outside of any specific unit test.

Reply:
All functionalities of a communicator should be verified in the corresponding unit tests for communicator.

Reply:
Ok I am fine with either approach, the main point is that it should be directly obvious that the test should fail if it is not run with sufficient number of ranks!